.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?
+- [ ] Have you verified that you are running the latest version of Nancy?
+- [ ] Have you verified if the problem exist in both `debug` and `release` mode?
+- [ ] Have you searched [open issues](https://github.com/NancyFx/Nancy/issues) to ensure it has not already been reported?
+
+### Description
+<!-- A description of the bug or feature -->
+
+### Steps to Reproduce
+<!-- List of steps, sample code, failing test or link to a project that reproduces the behavior -->
+
+### System Configuration
+<!--
+Tell us about the environment where you are experiencing the bug]
+
+- Nancy version

```

Comment:
I'm thinking that these shouldn't be comments, but actually something that the user should fill out. Something like

```
- Nancy version: <!-- Please specify Nancy version -->
- Nancy host
  - [ ] ASP.NET
  - [ ] OWIN
  - [ ] Self-Hosted
  - [ ] Other: <!-- Please specify the host -->
```